api/swagger.yml

```
@@ -494,6 +494,42 @@ paths:
+        200:
+          description: The rendered report (or restricted notice) HTML
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ReportPreviewResponse"
+        400:
+          $ref: "#/components/responses/BadRequest"
+        500:
+          $ref: "#/components/responses/Internal"
+      security:
+        - bearerAuth: [ ]
```

YAMLlint (1.37.1)

[error] too many spaces inside empty brackets

(brackets)